@rolyp I'm getting a failure from mypy : `ways_py/ways.py:12: error: Function is missing a type annotation for one or more arguments  [no-untyped-def]` which on googling looked like I needed to add function annotation (i.e. using `-> alt.Chart` in the function def), but seems to be still saying that after I've changed it so not sure why, any ideas?

--- a/ways_py/ways.py
+++ b/ways_py/ways.py
@@ -9,7 +9,7 @@ class Ways:
         df: pd.DataFrame = pd.DataFrame(columns=["x", "y"])
         return alt.Chart(df, title="Dummy Chart").properties(width=600, height=500).mark_point()
 
-    def altair_meta_hist(dataframe, column, bin, color) -> alt.Chart:
+    def altair_meta_hist(dataframe: pd.DataFrame, column: str, bin: alt.Bin, color: alt.Color) -> alt.Chart:
         """Altair metavisualisation histogram.
 
         Plot a histogram metavisualisation for a plot with matching

